frontend/src/components/page-events/eventCard.js

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import Card from "../shared/card";
+import prayerGatheringImg from "../../images/prayer-gathering.png";
+import mapPinLogo from "../../images/icons/map-pin.svg";
+
+const EventCard = ({ title, date, location, img, description }) => {
+  const attributes = (
+    <div>
+      <div className="flex items-center gap-1">
+        <img className="w-4 h-4 mb-0" src={mapPinLogo} alt="Map" />
```

Comment:
Recent changes by @graceschae  she added StaticImage  Component from Gatsby: 
```suggestion
        <StaticImage className="w-4 h-4 mb-0" src="../../images/icons/map-ping.svg" alt="Map" />
```